mir_eval/separation.py

```diff
@@ -107,7 +107,9 @@ def validate(reference_sources, estimated_sources):
                                                   MAX_SOURCES))
 
 
-def bss_eval_sources(reference_sources, estimated_sources):
+def bss_eval_sources(
+    reference_sources, estimated_sources, compute_permutation=True
+):
```

Comment:
It does (see https://www.python.org/dev/peps/pep-0008/#indentation) and I like new new lines a lot...
Anyway, I changed it now so that the indentation style matches the rest of the package 😄